src/components/backend-ai-session-list.ts

```diff
@@ -1347,20 +1359,31 @@ export default class BackendAiSessionList extends BackendAIPage {
             </div>
             <div class="layout horizontal center flex">
               <div class="layout horizontal configuration">
-                <template is="dom-if" if="[[item.gpu_slot]]">
-                  <wl-icon class="fg green indicator">view_module</wl-icon>
-                  <span>[[item.gpu_slot]]</span>
+                <template is="dom-if" if="[[item.cuda_gpu_slot]]">
+                  <img class="indicator-icon fg green" src="/resources/icons/file_type_cuda.svg" />
+                  <span>[[item.cuda_gpu_slot]]</span>
                   <span class="indicator">GPU</span>
                 </template>
                 <template is="dom-if" if="[[!item.gpu_slot]]">
-                  <template is="dom-if" if="[[item.fgpu_slot]]">
-                    <wl-icon class="fg green indicator">view_module</wl-icon>
-                    <span>[[item.fgpu_slot]]</span>
+                  <template is="dom-if" if="[[item.cuda_fgpu_slot]]">
+                    <img class="indicator-icon fg green" src="/resources/icons/file_type_cuda.svg" />
+                    <span>[[item.cuda_fgpu_slot]]</span>
                     <span class="indicator">GPU</span>
                   </template>
                 </template>
-                <template is="dom-if" if="[[!item.gpu_slot]]">
-                  <template is="dom-if" if="[[!item.fgpu_slot]]">
+                <template is="dom-if" if="[[item.rocm_gpu_slot]]">
+                  <wl-icon class="fg green indicator">view_module</wl-icon>
+                  <img class="indicator-icon fg green" src="/resources/icons/ROCm.png" />
+                  <span>[[item.rocm_gpu_slot]]</span>
+                  <span class="indicator">GPU</span>
+                </template>
+                <template is="dom-if" if="[[item.tpu_slot]]">
+                  <wl-icon class="fg green indicator">view_module</wl-icon>
+                  <span>[[item.tpu_slot]]</span>
+                  <span class="indicator">TPU</span>
+                </template>
+                <template is="dom-if" if="[[!item.cuda_gpu_slot]]">
+                  <template is="dom-if" if="[[!item.cuda_fgpu_slot]]">
                     <wl-icon class="fg green indicator">view_module</wl-icon>
```

Comment:
Perhaps ROCM gpu should also be considered here to indicate no GPUs?

```html
<template is="dom-if" if="[[!item.rocm_gpu_slot]]">
```

Reply:
Fixed.